_maps/star_trek.json

```diff
@@ -3,12 +3,11 @@
     "map_path": "map_files/StarTrek13",
     "map_file": ["overmap.dmm",
 				"overmap_s2.dmm",
-				"overmap.dmm",
```

Comment:
wait it had two of these fuck..... Oh well it probably never spawned anything on the second overmap.dmm